projects/igniteui-angular/src/lib/grids/grid-public-row.ts

```diff
@@ -51,6 +52,16 @@ abstract class BaseRow implements RowType {
             this.grid.crudService.row.id === this.key;
     }
 
+    /** Gets the validation errors if any.
+    * ```typescript
+    * let errors = row.errors;
+    * ```
+    */
+    public errors(): ValidationErrors {
+        const formGroup = this.grid.validation.getFormGroup(this.key);
```

Comment:
This requires additional check. `FormGroup` might not be created yet.

Reply:
@dkamburov There is a check on the line below:
```
return formGroup?.errors;
```
If there's no formGroup it will return undefined.